subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/DefaultTaskContainerTest.groovy

```diff
@@ -1659,6 +1651,4 @@ class DefaultTaskContainerTest extends AbstractPolymorphicDomainObjectContainerS
     interface CustomTask extends TaskInternal {}
 
     interface MyCustomTask extends CustomTask {}
-
-    interface AnotherCustomTask extends TaskInternal {}
```

Comment:
❔ Any idea what that task has been used for?

Reply:
The tests that used this additional class were removed a long time ago, but the class has been left hanging